db/python/tables/analysis.py

```diff
@@ -431,16 +433,16 @@ async def get_sample_cram_path_map_for_seqr(
             filters.append('JSON_VALUE(a.meta, "$.sequencing_type") ' + seq_check)
 
         if participant_ids:
-            filters.append('p.id IN :pids')
+            filters.append('peid.participant_id IN :pids')
```

Comment:
This field needs to be changed to `participant_id` (as per the new table) here too. And a minor renaming of the table alias.